src/main/java/org/jahia/modules/serversettings/flow/ImportInfo.java

```diff
@@ -82,10 +82,10 @@
     private String templates;
     private String type;
     private ValidationResults validationResult;
-    private int originatingBuildNumber;
+    private String originatingBuildNumber;
```

Comment:
Do we really need to keep this if it's not used anywhere anymore ?

Reply:
Actually you are correct, it's only used in one spot so I will remove it at all